FamiliyDoctor/FamiliyDoctor/src/main/java/com/familydoctor/controllers/MedicalRecordController.java

```diff
@@ -0,0 +1,44 @@
+package com.familydoctor.controllers;
+
+import com.familydoctor.models.DTO.MedicalRecordDTO;
+import com.familydoctor.services.MedicalRecordService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class MedicalRecordController {
```

Comment:
For the DoctorController you have the mapping `@RequestMapping("/api/doctors")`. Please be aware of consistency and to the same `/api/doctors...` for this or any other controllers you have.